### PR TITLE
Student Profile Page

### PR DESCRIPTION
**Pull request by Saarim-Salim**
init student profile page.

### CODE REVIEW

File: app/[locale]/profile/tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Quick Send' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => setActiveTab('Quick Send')}
+        >
+          <TbFileSearch size="1.75rem" />
+
+          <span>QUICK SEND</span>
+        </button>
+      </nav>
+      <button className="mt-5 justify-center text-primary-700">LOG OUT</button>
```
Comment (GetPsyched) [inline app/[locale]/profile/tabs.tsx:93]: See this for logout:
https://github.com/nkss-dev/node-modules/blob/1d88b5974cd688caa7ca8cc5ba5047c5e6db109e/app/breadcrumb.tsx#L181

File: app/[locale]/profile/tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
+'use client';
+import Image from 'next/image';
```
Comment (GetPsyched) [inline app/[locale]/profile/tabs.tsx:2]:
```suggestion
'use client';

import Image from 'next/image';
```

File: app/[locale]/profile/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  params: { locale },
+}: {
+  children: React.ReactNode;
+  params: { locale: string };
+}) => {
+  const session = await getServerAuthSession();
+
+  return session ? (
+    <section>
+      <div className="bg-gray-100 mt-11 flex h-full gap-2 p-16">
```
Comment (GetPsyched) [inline app/[locale]/profile/layout.tsx:17]: Why do we have a nested section and div?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/[locale]/profile/layout.tsx (hunk 3 above)
Comment (Saarim-Salim) [reply app/[locale]/profile/layout.tsx:17]: Just that the code was refactored multiple times and moved between files, might've missed out on this. The nested div is the parent container of tabs and the displayed container. Changing section tag to angle brackets.

File: app/[locale]/profile/tabs.tsx (hunk 1 above)
Comment (Saarim-Salim) [reply app/[locale]/profile/tabs.tsx]: pushed.

File: app/[locale]/profile/layout.tsx [hunk 4]
```diff
@@ -14,9 +14,9 @@ const ProfileLayout = async ({
   const session = await getServerAuthSession();
 
   return session ? (
-    <section>
+    <>
       <div className="bg-gray-100 mt-11 flex h-full gap-2 p-16">{children}</div>
-    </section>
+    </>
```
Comment (GetPsyched) [inline app/[locale]/profile/layout.tsx:19]: Just remove the tags entirely, they serve no purpose. You only need them when wrapping multiple elements but don't want to add a wrapper tag.

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: app/[locale]/profile/layout.tsx (hunk 4 above)
Comment (Saarim-Salim) [reply app/[locale]/profile/layout.tsx:19]: alright, shifting wrapper component to page.tsx

Comment (Saarim-Salim) [reply app/[locale]/profile/layout.tsx]: pushed

File: app/[locale]/profile/tabs.tsx [hunk 5]
```diff
@@ -139,6 +139,8 @@ export default function NavTabs({ activeTab }: NavTabsProps) {
             <DropdownMenuSeparator />
             {activeTab !== 'PersonalDetails' && (
               <DropdownMenuItem>
+                {' '}
+                r
```
Comment (GetPsyched) [inline app/[locale]/profile/tabs.tsx:143]: ?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

**Review by GetPsyched — APPROVED** [review]

**Pull request merged**